src/slack-send.js

```diff
@@ -128,9 +131,11 @@ module.exports = async function slackSend(core) {
 
       try {
         await axios.post(webhookUrl, payload, axiosOpts);
+        console.log(JSON.stringify(payload));
```

Comment:
This might not be desired behavior for a successful `POST` since it could expose values in the action logs. Not so against it, but I'm wondering if there's a strong reason for including it?